src/reference/asciidoc/aggregator.adoc

```diff
@@ -347,214 +334,157 @@ Below you can see an example of an aggregator.
 <bean id="correlationStrategyBean" class="sample.PojoCorrelationStrategy"/>
 ----
 
-<1> The id of the aggregator is _Optional_.
-
-
-
-<2> Lifecycle attribute signaling if aggregator should be started during Application Context startup.
-_Optional (default is 'true')_.
-
-
-
-<3> The channel from which where aggregator will receive messages.
-_Required_.
-
-
-
-<4> The channel to which the aggregator will send the aggregation results.
-_Optional (because incoming messages can specify a reply channel themselves via 'replyChannel' Message Header)_.
-
-
-
-<5> The channel to which the aggregator will send the messages that timed out (if `send-partial-result-on-expiry` is _false_).
-_Optional_.
-
-
-
+<1> The id of the aggregator is optional.
+<2> Lifecycle attribute signaling whether the aggregator should be started during application context startup.
+Optional (the default is 'true').
+<3> The channel from which where aggregator receives messages.
+Required.
+<4> The channel to which the aggregator sends the aggregation results.
+Optional (because incoming messages can themselves specify a reply channel in the 'replyChannel' message header).
+<5> The channel to which the aggregator sends the messages that timed out (if `send-partial-result-on-expiry` is `false`).
+Optional.
 <6> A reference to a `MessageGroupStore` used to store groups of messages under their correlation key until they are complete.
-_Optional_, by default a volatile in-memory store.
-
-
-
-<7> Order of this aggregator when more than one handle is subscribed to the same DirectChannel (use for load balancing purposes).
-_Optional_.
-
-
-
-<8> Indicates that expired messages should be aggregated and sent to the 'output-channel' or 'replyChannel' once their containing `MessageGroup` is expired (see `MessageGroupStore.expireMessageGroups(long)`).
-One way of expiring `MessageGroup` s is by configuring a `MessageGroupStoreReaper`.
-However `MessageGroup` s can alternatively be expired by simply calling `MessageGroupStore.expireMessageGroups(timeout)`.
-That could be accomplished via a Control Bus operation or by simply invoking that method if you have a reference to the `MessageGroupStore` instance.
-Otherwise by itself this attribute has no behavior.
-It only serves as an indicator of what to do (discard or send to the output/reply channel) with Messages that are still in the `MessageGroup` that is about to be expired.
-_Optional_.
-_Default - false_.
-*NOTE:* This attribute is more properly `send-partial-result-on-timeout` because the group may not actually expire if
+Optional.
+By default, it is a volatile in-memory store.
+// TODO How would one change it?
```

Comment:
Well, everything is about beans in Spring.
The sentence like "A reference to a `MessageGroupStore`" should do the trick and make it obvious that you need to configure a bean on the matter and point its bean name here.
Yeah... we can point from here to the `[[message-store]]` chapter though...